common/ledger/blkstorage/blockindex.go

```diff
@@ -248,6 +251,41 @@ func (index *blockIndex) getTXLocByBlockNumTranNum(blockNum uint64, tranNum uint
 	return txFLP, nil
 }
 
+func (index *blockIndex) exportUniqueTxIDs(writer io.Writer) error {
```

Comment:
will the writer be a buffer that helps in computing the hash before appending the data to a snapshot file?